packages/shared/src/components/post/collection/CollectionPillSources.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import Pill from '../../Pill';
+import { SourceAvatar, SourceAvatarProps } from '../../profile/source';
+import { ProfilePictureGroup } from '../../ProfilePictureGroup';
+
+interface CollectionPillSourcesProps {
+  className?: string;
+  sources: SourceAvatarProps['source'][];
+  totalSources: number;
+  shouldShowSources: boolean;
+}
+export const CollectionPillSources = ({
+  className,
+  sources,
+  totalSources,
+  shouldShowSources,
+}: CollectionPillSourcesProps): ReactElement => {
+  return (
+    <div className={classNames(className, 'flex relative flex-row gap-2')}>
+      <div className="relative">
+        {shouldShowSources && (
```

Comment:
Why do we use state for this?
Feels like we could easily use the group-hover for this now with the refactor?